andela/travel_tool_front
# #160710724 Pay Technical Debt for Full Guest House Details

#### What does this PR do?
- Adds tests for all components, actions and action creators, sagas and API service
- Adds `Today` button to the timeline for the user to navigate back to the current period in the calendar
- Resolves problem with hidden trip details on the timeline when user clicks to the far right of the timeline
- Add `Origin` information to the trip details
- Add mobile responsiveness to the full guesthouse details page and timeline

#### Description of Task to be completed?
- Have all functionalitites tested and desirable behaviours safeguarded by the tests
- Impement a `Go back to today` feature
- Resolve issue with the trip details sliding out of visibility when a user clicks on a trip bar at the right edge of the timeline container
- Add information about the requester's centre of origin to the trip details

#### How should this be manually tested?
Please see guide on [this](https://github.com/andela/travel_tool_front/pull/161) parent Pull Request on how to set up the project locally
- On the timeline, navigate the period away from the current date and finally click on `Today`. You'd be taken back to the current period in time; week, month or year
- Check the mobile resposiveness of the page and the timeline

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#160710724](https://www.pivotaltracker.com/story/show/160710724)

#### Screenshots (if appropriate)
#### Questions:
